docs/staking-and-delegation/stakers-btcli-guide.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Staker's Guide to `BTCLI`"
+---
+
+# Staker's Guide to `BTCLI`
+
+This page prepares the reader for managing TAO and alpha stake using `btcli` in a secure manner.
+
+For general coverage of `btcli` security and usage considerations across persona, see: [Bittensor CLI: Permissions Guide](../btcli-permissions)
+
+## Intro
+
+Stakers enter value into the Bittensor network by acquiring TAO and staking or *delegating* it to validators to support their work. As validators extract emissions, a certain percentage goes back to stakers.
+
+Stakers must be familiar with operations related to managing the TAO and staked alpha tokens in their Bittensor wallet balances.
+
+Account balances are public information, and can be viewed *without* using a coldkey, even in an insecure environment. However, any account operation that **changes the state** of the Bittensor chain, such as a balance transfer or staking operation, must be signed with your wallet's coldkey private key.
+
+Performing these functions requires using a **coldkey**, and hence must be performed in a [**highly secure environment**](../getting-started/coldkey-hotkey-security) for any wallet connected to real (mainnet) TAO balance. A leak of your coldkey can lead to a catastrophic loss of funds.
+
+Any operation can be practiced against Bittensor testnet using throw-away keys *not* connected to your real TAO balances on mainnet.
+
+:::tip
+Stakers only need a coldkey. Unless you plan to mine, validate, or participate in governance, you do not need a hotkey.
+:::
+
+See:
+- [Staking/Delegation Overview](./delegation.md)
+- [Bittensor CLI: Permissions Guide](../btcli-permissions)
+- [Wallets, Coldkeys and Hotkeys in Bittensor](../getting-started/wallets)
+- [Coldkey and Hotkey Workstation Security](../getting-started/coldkey-hotkey-security)
+
+## Requirements for wallet, balance and staking functions
+
+### Wallets
+  - View balances and stake&mdash;only coldkey public key is required (permissionless)
+  - Create or generate a new coldkey (coldkey required).
+  - transfer TAO
+  - Transfer TAO from your coldkey to another address. Requires a coldkey signature (secure environment).
```

Comment:
Are the the same? If not, what is the difference?

Reply:
yup, just slop, ty!